lib/open_food_network/customers_report.rb

```diff
@@ -60,7 +60,11 @@ def filter(orders)
     def filter_to_supplier(orders)
       if params[:supplier_id].to_i > 0
         orders.select do |order|
-          order.line_items.includes(:product).references(:product).where("spree_products.supplier_id = ?", params[:supplier_id].to_i).count > 0
+          order.line_items.includes(:product)
+            .where("spree_products.supplier_id = ?", params[:supplier_id].to_i)
```

Comment:
it should be possible to do `where(spree_products: { supplier_id: params[:supplier_id].to_i)` as well, but again, just a minor comment.